subworkflows/local/prepare_input.nf

```diff
@@ -7,15 +7,16 @@ include { SAMTOOLS_FASTQ } from "../../modules/nf-core/modules/samtools/fastq/ma
 /*
     Input format descriptions
 
-    Simple: CSV
+    Simple: TSV ( splitCsv cannot handle embedded commas )
     Flexible: YAML
     ( See Notes: at bottom for implementation details )
 
     Simple Format:
 
-    ```csv
-    sample_id,data_type,paths
-    sample_A,HiFi,/path/to/hifi
+    ```tsv
+    sample_id   data_type   sequences
+    sample_A    HiFi   /path/to/hifi/reads.bam
+    sample_A    Illumina    /path/to/illumina/reads{1,2}.fastq.gz
```

Comment:
I have the same concern, whether `/path/to/illumina/reads{1,2}.fastq.gz` is the only way to specify paired end reads. It just assumes a certain naming scheme. I am not sure how that would work in the yml, but I assume multiple files can be specified?

With the current implementation, would it be possible to also use a list entry for PE illumina in the TSV? E.g.
```tsv
sample_id   data_type   sequences
sample_A    Illumina    ["/path/to/illumina/reads1.fastq.gz","/path/to/illumina/reads2.fastq.gz"]
```
If yes, would be great to have documented that.

Reply:
I think `file` only accepts a valid string format for a path ( that can include globs). I don't think an array is possible like so. I was also hoping we could just use the cardinality to specify paired end vs single end as files belonging to the same sample will be grouped on exit. Is there something you had in mind about the specification of the reads?

Reply:
Yes I think you are right that it wont work as above.

> Is there something you had in mind about the specification of the reads?

I liked the sample sheet input in tsv / csv typically to specify forward and reverse file names, without globs or such. But I can accept the way you did it.
For PE it would be nice to maybe allowing comma separated entries or any other way specifying two paths. But maybe thats just me, than its not worth adding ;)

Reply:
Would you prefer the notation of `forward` and `reverse` instead of `read1`/`read2`?

Reply:
Hm, read1/read2 seems more generic. forward/reverse implicates a direction, e.g. stranded lib prep, amplicon seq. But the usual genome seq isn't stranded, so read1/read2 might be better? But the notation is to me not too important.